include/deal.II/matrix_free/operators.h

```diff
@@ -1512,7 +1606,7 @@ namespace MatrixFreeOperators
                     const std::pair<unsigned int,unsigned int>  &cell_range) const
   {
     typedef typename Base<dim,VectorType>::value_type Number;
-    FEEvaluation<dim,fe_degree,n_q_points_1d,n_components,Number> phi (data);
+    FEEvaluation<dim,fe_degree,n_q_points_1d,n_components,Number> phi (data, this->selected_rows[0]);
     for (unsigned int cell=cell_range.first; cell<cell_range.second; ++cell)
```

Comment:
You are right, we can only use `MassOperator` and `LaplaceOperator` for a single block, otherwise we would need multiple `FEEvaluation` objects. I will add a comment.